frontend/src/metabase/parameters/components/widgets/StringInputWidget/StringInputWidget.tsx

```diff
@@ -55,25 +55,27 @@ function StringInputWidget({
           value={unsavedArrayValue}
           onChange={setUnsavedArrayValue}
           placeholder={placeholder}
-          options={OPTIONS}
+          options={[]}
```

Comment:
passing new array can cause unnecessary re-renderings, but it shouldn't be a big problem

Reply:
That is a good catch actually! Thanks.